draft-ietf-quic-transport.md

```diff
@@ -2207,14 +2212,15 @@ same address family for the server.
 
 # Connection Termination {#termination}
 
-Connections should remain open until they become idle for a pre-negotiated
-period of time.  A QUIC connection, once established, can be terminated in one
-of three ways:
+An established QUIC connection can be terminated in one of three ways:
 
 * idle timeout ({{idle-timeout}})
 * immediate close ({{immediate-close}})
 * stateless reset ({{stateless-reset}})
 
+In addition, an endpoint MAY drop a connection and discard all connection state
+without informing the peer if it is unable to reach the peer.
```

Comment:
How about "repeatedly unable to reach the peer."  Also, in this case, a connection close packet should still be sent in case the peer can receive packets, but packets it sends cannot reach you.

Also is 'reach the peer' obvious wording?

Reply:
I was wondering about this too, but assumed there was a clear definition in an earlier section. Even if you do not get a response, the peer might be able to hear a close. In that context, MAY seems fine.

Reply:
Maybe instead:

> An endpoint MAY discard connection state if it does not have a validated path on which it can send packets.